quint/src/runtime/testing.ts

```diff
@@ -118,11 +120,24 @@ export function compileAndTest(
         for (; nsamples <= options.maxSamples; nsamples++) {
           // record the seed value
           seed = options.rng.getState()
-          // run the test
+          // reset the trace
           recorder.onRunCall()
```

Comment:
Perhaps this comment belongs on the next line
```suggestion
          recorder.onRunCall()
          // reset the trace
```